ui/styles/components/views.less

```diff
@@ -1,10 +1,13 @@
 @import "../constants/colors";
 
-.simple img, .special{
+.simple img, .special {
   max-width: 100%;
   max-height: 100%;
   display: table;
   margin: 0 auto;
+  .clearfix {
```

Comment:
Can't you target a different element here? The `.clearfix` trick is tied to `float` so we shouldn't depend on it for applying margins.